src/Storage/Storage/Blob/Cmdlet/NewAzureStorageContainerSasToken.cs

```diff
@@ -151,7 +157,7 @@ public override void ExecuteCmdlet()
             }
 
             //Create SAS builder
-            BlobSasBuilder sasBuilder = SasTokenHelper.SetBlobSasBuilder_FromContainer(container, identifier, this.Permission, this.StartTime, this.ExpiryTime, this.IPAddressOrRange, this.Protocol, this.EncryptionScope);
+            BlobSasBuilder sasBuilder = SasTokenHelper.SetBlobSasBuilder_FromContainer(container, identifier, this.Permission, this.StartTime, this.ExpiryTime, this.IPAddressOrRange, this.Protocol, this.EncryptionScope, this.DelegationObjectID);
```

Comment:
in new line 133, should also remind user UD sas is gen when `Channel.StorageContext.Track2OauthToken != null`
And should report error when "DelegatedUserObjectId" be specified when input Storage Context is NOT OAuth based.